scripts/release/rpm/azure-cli.spec

```diff
@@ -26,6 +26,8 @@ Release:        %{release}
 Url:            https://docs.microsoft.com/cli/azure/install-azure-cli
 BuildArch:      x86_64
 Requires:       %{python_cmd}
+Prefix:         /usr
+Prefix:         /etc
```

Comment:
Add `Prefix` to support [relocatable package](http://ftp.rpm.org/api/4.4.2.2/relocatable.html).